pkg/quota/generic/evaluator.go

```diff
@@ -167,9 +167,6 @@ func CalculateUsageStats(options quota.UsageStatsOptions,
 // that associates usage of the specified resource based on the number of items
 // returned by the specified listing function.
 type objectCountEvaluator struct {
-	// allowCreateOnUpdate if true will ensure the evaluator tracks create
-	// and update operations.
-	allowCreateOnUpdate bool
```

Comment:
just curious why this is removed from the quota controller?

Reply:
@nikhita We will only check `CREATE` admission for create-on-update request, so we don't need to consider `allowCreateOnUpdate` anymore in evaluator from now on.😃